tests/notifications/NotificationsManager.test.ts

```diff
@@ -22,7 +22,7 @@ import * as notificationsErrors from '@/notifications/errors';
 import * as vaultsUtils from '@/vaults/utils';
 import * as nodesUtils from '@/nodes/utils';
 import * as keysUtils from '@/keys/utils';
-import * as testUtils from '../utils';
+import * as testUtils from '../utils/utils';
```

Comment:
This should be `import * as testUtils from '../utils';`. No need to specify the internal module. The `index.ts` takes care of it.